### PR TITLE
Feat: New runtimes

### PR DESCRIPTION
**Pull request by Meldiron**
## What does this PR do?

Corrects base images, and adds newly supported runtimes by OPR

## Test Plan

None

## Related PRs and Issues

x

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

File: src/Runtimes/Runtimes.php [hunk 1]
```diff
@@ -20,95 +20,104 @@ public function __construct(string $version = '')
...
-        $java->addVersion('21.0', 'openjdk/21-jdk-slim', 'openruntimes/java:'.$this->version.'-21.0', [System::X86, System::ARM64, System::ARMV7, System::ARMV8]);
+        $java->addVersion('8.0', 'eclipse-temurin:8-jdk-jammy', 'openruntimes/java:'.$this->version.'-8.0', [System::X86, System::ARM64, System::ARMV7, System::ARMV8]);
+        $java->addVersion('11.0', 'eclipse-temurin:11-jdk-jammy', 'openruntimes/java:'.$this->version.'-11.0', [System::X86, System::ARM64, System::ARMV7, System::ARMV8]);
+        $java->addVersion('17.0', 'eclipse-temurin:17-jdk-jammy', 'openruntimes/java:'.$this->version.'-17.0', [System::X86, System::ARM64, System::ARMV7, System::ARMV8]);
+        $java->addVersion('18.0', 'eclipse-temurin:18-jdk-jammy', 'openruntimes/java:'.$this->version.'-18.0', [System::X86, System::ARM64, System::ARMV7, System::ARMV8]);
+        $java->addVersion('21.0', 'eclipse-temurin:21-jdk-jammy', 'openruntimes/java:'.$this->version.'-21.0', [System::X86, System::ARM64, System::ARMV7, System::ARMV8]);
+        $java->addVersion('22.0', 'eclipse-temurin:22-jdk-jammy', 'openruntimes/java:'.$this->version.'-22.0', [System::X86, System::ARM64, System::ARMV7, System::ARMV8]);
```
Comment (ItzNotABug) [inline src/Runtimes/Runtimes.php:91]: Images seem different here too, what changed?

**Review by ItzNotABug — APPROVED** [review]
LGTM, just a few comments

File: src/Runtimes/Runtimes.php [hunk 2]
```diff
@@ -20,95 +20,104 @@ public function __construct(string $version = '')
...
+        $python->addVersion('3.9', 'python:3.9.20-alpine3.20', 'openruntimes/python:'.$this->version.'-3.9', [System::X86, System::ARM64, System::ARMV7, System::ARMV8]);
+        $python->addVersion('3.10', 'python:3.10.15-alpine3.20', 'openruntimes/python:'.$this->version.'-3.10', [System::X86, System::ARM64, System::ARMV7, System::ARMV8]);
+        $python->addVersion('3.11', 'python:3.11.10-alpine3.20', 'openruntimes/python:'.$this->version.'-3.11', [System::X86, System::ARM64, System::ARMV7, System::ARMV8]);
+        $python->addVersion('3.12', 'python:3.12.6-alpine3.20', 'openruntimes/python:'.$this->version.'-3.12', [System::X86, System::ARM64, System::ARMV7, System::ARMV8]);
         $this->runtimes['python'] = $python;
 
         $pythonML = new Runtime('python-ml', 'Python (ML)', 'sh helpers/server.sh');
-        $pythonML->addVersion('3.11', 'continuumio/miniconda3:24.1.2-0', 'openruntimes/python-ml:'.$this->version.'-3.11', [System::X86, System::ARM64]);
+        $pythonML->addVersion('3.11', 'python:3.11.10-bookworm', 'openruntimes/python-ml:'.$this->version.'-3.11', [System::X86, System::ARM64]);
```
[the remark this answers is not in the document]
Comment (Meldiron) [reply src/Runtimes/Runtimes.php:55]: No idea, this was probably outdated for quite some time. But it matches OPR: https://github.com/open-runtimes/open-runtimes/blob/main/runtimes/python/versions/ml-3.11/Dockerfile#L2

File: src/Runtimes/Runtimes.php (hunk 1 above)
Comment (Meldiron) [reply src/Runtimes/Runtimes.php:91]: No idea, this was probably outdated for quite some time. But it matches OPR: https://github.com/open-runtimes/open-runtimes/blob/main/runtimes/java/versions/11.0/Dockerfile#L2

**Review by ItzNotABug — APPROVED** [review]

File: src/Runtimes/Runtimes.php [hunk 3]
```diff
@@ -20,95 +20,104 @@ public function __construct(string $version = '')
...
-        $kotlin->addVersion('1.8', 'openjdk/19-jdk-slim', 'openruntimes/kotlin:'.$this->version.'-1.8', [System::X86, System::ARM64, System::ARMV7, System::ARMV8]);
-        $kotlin->addVersion('1.9', 'openjdk/19-jdk-slim', 'openruntimes/kotlin:'.$this->version.'-1.9', [System::X86, System::ARM64, System::ARMV7, System::ARMV8]);
+        $kotlin->addVersion('1.6', 'eclipse-temurin:18-jdk-alpine', 'openruntimes/kotlin:'.$this->version.'-1.6', [System::X86, System::ARM64, System::ARMV7, System::ARMV8]);
+        $kotlin->addVersion('1.8', 'eclipse-temurin:19-jdk-alpine', 'openruntimes/kotlin:'.$this->version.'-1.8', [System::X86, System::ARM64, System::ARMV7, System::ARMV8]);
+        $kotlin->addVersion('1.9', 'eclipse-temurin:19-jdk-alpine', 'openruntimes/kotlin:'.$this->version.'-1.9', [System::X86, System::ARM64, System::ARMV7, System::ARMV8]);
+        $kotlin->addVersion('1.9', 'eclipse-temurin:19-jdk-alpine', 'openruntimes/kotlin:'.$this->version.'-1.9', [System::X86, System::ARM64, System::ARMV7, System::ARMV8]);
```
Comment (abnegate) [inline src/Runtimes/Runtimes.php:105]: Duplicate

**Review by abnegate — APPROVED** [review]

**Review by TorstenDittmann — APPROVED** [review]

File: src/Runtimes/Runtimes.php (hunk 3 above)
Comment (Meldiron) [reply src/Runtimes/Runtimes.php]: Nice catch! Fixed, it was meant to be 2.0

**Pull request merged**